Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:9-alpine
+FROM node:18.14.0-alpine3.17
```

Comment:
thats a big node version jump, are we sure everything still works as expected?

Reply:
I built the image and ran it against the sample codebase inside the `test/` directory. Everything seems to be working fine, issues are appearing when they have to! 👍